x/ibc/03-connection/keeper/handshake_test.go

```diff
@@ -84,7 +84,7 @@ func (suite *KeeperTestSuite) TestConnOpenTry() {
 		//check connection state
 		conn, existed := suite.app.IBCKeeper.ConnectionKeeper.GetConnection(suite.ctx, testConnectionID1)
 		suite.True(existed)
-		suite.Equal(connection.TRYOPEN, conn.State)
+		suite.Equal(connection.TRYOPEN.String(), conn.State.String(), "invalid connection state")
```

Comment:
Why was this change necessary?

Reply:
I made that change bc otherwise the error message from the tests was not helpful